.semaphore/release.yml

```diff
@@ -24,6 +24,7 @@ blocks:
           commands:
             - nvm use
             - yarn
+            - yarn build-esm
```

Comment:
I don't know if we want to include this build process in the current `prepare` stage in dev, it will make the entire yarn task slower, so only included it in the CI release process

Reply:
is the CI who is responsible for the release in npm correct?

Reply:
Yes, it just if we want to release manually, we need to remember to run this command as well